### PR TITLE
LPS-107010 portal-search-tuning-rankings-web: Fix UX for activating rankings

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-107010

@jonmak08

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

**Comment by oliv-yu** [comment]
ci:test:relevant:norebase

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/portlet/action/ValidateRankingMVCResourceCommand.java [hunk 1]
```diff
@@ -205,27 +219,39 @@ public ValidateRankingMVCResourceRequest(
 			ResourceRequest resourceRequest) {
 
 			_indexName = ParamUtil.getString(resourceRequest, "index-name");
+			_inactive = ParamUtil.getBoolean(resourceRequest, "inactive");
 			_resultsRankingUid = ParamUtil.getString(
 				resourceRequest, "resultsRankingUid");
 
 			_aliases = Arrays.asList(
 				ParamUtil.getStringValues(resourceRequest, PARAM_ALIASES));
+			_queryString = ParamUtil.getString(resourceRequest, PARAM_KEYWORDS);
```
Comment (thektan) [inline modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/portlet/action/ValidateRankingMVCResourceCommand.java]: Let's remove the line break and alphabetize these.

Comment (thektan) [reply modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/portlet/action/ValidateRankingMVCResourceCommand.java]: Let's also remove `PARAM_ALIASES` and `PARAM_KEYWORDS` and use the direct String since they aren't being used anywhere else. 

```java
_queryString = ParamUtil.getString(resourceRequest, "keywords");
```

File: modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -5,7 +5,7 @@ add-result=Add Result
 add-results-help=Add results that are not normally returned by the main search query.
 alias=Alias
 aliases=Aliases
-aliases-must-be-unique-across-all-rankings.-the-following-aliases-already-exist-x=Aliases must be unique across all rankings. The following aliases already exist: {0}.
+active-search-queries-and-aliases-must-be-unique-across-all-rankings.-the-following-ones-already-exist-x=Active search queries and aliases must be unique across all rankings. The following ones already exist: {0}.
```
Comment (thektan) [inline modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/content/Language.properties]: Alphabetize

**Comment by oliv-yu** [comment]
ci:test:search:norebase

**Comment by thektan** [comment]
@oliv-yu When you get the chance, could you take a look at this again? It seems like there's an edge case that doesn't work:
1) Create active ranking with query "test"
2) Create inactive ranking with alias "test"
3) Edit inactive ranking: activate and remove alias "test"

Page goes to an error page rather than successfully saving.

I looked at the params being passed to the `/results_ranking/validate` endpoint and they seem good so you'll probably have to set a breakpoint and see what's happening in the validate. No errors in tomcat logs either. 😕

**Comment by thektan** [comment]
Pull request submitted to https://github.com/thektan/liferay-portal/pull/140.

**Comment by thektan** [comment]
@oliv-yu Thanks for making those changes! Looks great!
